transit_odp/publish/forms/seasonal_services.py

```diff
@@ -190,3 +198,72 @@ def clean(self):
         if start > end:
             raise ValidationError("Start date must be earlier than end date")
         return cleaned_data
+
+
+class SeasonalServiceEditDateForm(GOVUKModelForm):
```

Comment:
I think you can loose a lot of these definitions by inheriting from SeasonalServiceEditRegistrationCodeDateForm and overriding layout

Reply:
I am not sure there is a `clean_registration_code` in `SeasonalServiceEditRegistrationCodeDateForm`.
If I inherit from there I get an error to insert the registration code